src/components/Matches.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
+class Matches extends Component {
+  state = {  }
```

Comment:
State is duplicaat van state in `MatchesContainer`. Laten we deze `Matches` dom houden en uitsluitend verantwoordelijk zijn voor rendering.